GET_STARTED.md

```diff
@@ -0,0 +1,442 @@
+- CCF [Modules API reference](https://microsoft.github.io/CCF/main/js/ccf-app/modules.html)
+- CCF application get started repos
+  - [CCF application template](https://github.com/microsoft/ccf-app-template)
+  - [CCF application samples](https://github.com/microsoft/ccf-app-samples)
+
+## Supported Programing Languages
+
+Applications can be written in
+
+- TypeScript
+- JavaScript
+- C++
+- More languages support upcoming on 2023
+
+## Development environment
+
+- Development container and VSCode [![Open in VSCode](https://img.shields.io/static/v1?label=Open+in&message=VSCode&logo=visualstudiocode&color=007ACC&logoColor=007ACC&labelColor=2C2C32)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/ccf-app-samples)
+- Github codespace: [![Github codespace](https://img.shields.io/static/v1?label=Open+in&message=GitHub+codespace&logo=github&color=2F363D&logoColor=white&labelColor=2C2C32)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=496290904&machine=basicLinux32gb&devcontainer_path=.devcontainer.json&location=WestEurope)
+- Linux Machine ([Create a VM](https://learn.microsoft.com/en-us/azure/confidential-computing/quick-create-portal) and [Install ccf](https://microsoft.github.io/CCF/main/build_apps/install_bin.html))
+
+## <img src="https://user-images.githubusercontent.com/42961061/191275583-88e00f94-73aa-4d66-9786-047987eb9fa9.png" height=50px> </img> (JS/Typescript) Applications
+
+To test a ccf application you need go through the following steps:
+
+- Start a CCF Network with at least one node
+- Initialize the CCF network with at least one (active member - user), this is done through [Network Governance Proposals](https://microsoft.github.io/CCF/main/governance/proposals.html).
+- Create an application [deployment proposal](https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html)
+- Submit the app deployment proposal to the network and all members accept it through voting, This is a part of [Network Governance](https://microsoft.github.io/CCF/main/governance/proposals.html).
+- Open the CCF network for users
+- Start to test your application endpoints
+
+### Build Application
+
+The application building prerequisites [[CCF](#ccf-install), NodeJS and NPM] must be installed,
+all will be preinstalled if you are using devcontainer environment,
+otherwise you need to manually install.
+
+In the checkout of this repository:
+
+```bash
+cd banking-app
+npm install
+# build and generate the application bundle and deployment proposal
+make build
+ls
+cd ..
+
+# a dist folder is created with app bundle.
+```
+
+---
+
+### Testing your Application
+
+There are servral ways and tools which helping to test your application
+
+- [Sandbox.sh](#testing-using-sandboxsh)
+
+  - Build an initialized CCF network and deploy your app on top of it
```

Comment:
If you use the sandbox, you don't need to deploy your app on it do you? I thought you used --js-bundle